Source/Extensions/Blazorise.DataGrid/EventArguments/DataGridSortOrderChangedEventArgs.cs

```diff
@@ -0,0 +1,31 @@
+﻿#region Using directives
+using System.Collections.Generic;
+#endregion
+
+namespace Blazorise.DataGrid;
+
+/// <summary>
+/// Provides information about the DataGrid's sort order
+/// </summary>
+public class DataGridSortOrderChangedEventArgs
+{
+    /// <inheritdoc cref="DataGrid{TItem}.SortMode"/>
+    public DataGridSortMode SortMode { get; init; }
+
+    /// <summary>
+    /// A collection of <see cref="SortOrder"/> items. Contains all columns by which to sort.
+    /// </summary>
+    /// <remarks>Returns an empty list if no sorting is applied.</remarks>
+    public IReadOnlyCollection<DataGridSortInfo> SortOrder { get; init; }
```

Comment:
I don't get this recurring `SortOrder` naming. Do you mean it's just regular info on how a column should be sorted, and not related to the order? In which case the naming `SortInfo` or `SortDefinition` or something else more general seems better?

Reply:
I guess I am using the wrong term :-( . 

This event contains a list of all columns in their sort order and sort direction (ascending, descending). It also contains the column's `Field` name and (if available) the `SortField` name.

Reply:
Fixed naming in 8bdd62859a36e46ebe0cf812f8f6f3e0e4c13544